tests/repositories/test_installed_repository.py

```diff
@@ -43,6 +47,7 @@ def test_load(mocker):
             {"remote.origin.url": "git@github.com:sdispater/pendulum.git"},
         ],
     )
+    mocker.patch("site.addpackage", side_effect=addpackage)
```

Comment:
Unsure why this is required. If code works as expected, the discovered `site-packages` directory would be `tests/repositories/fixtures/installed/lib/python3.7/site-packages/`. This contains a file `editable.pth`, which would be read and hence loading its content `/path/to/editable` as the value for `editable.source_url`.